styles.css

```diff
@@ -6,7 +6,6 @@ img {
   height: 100px;
   width: auto;
   transition-duration: 0.3s;
-  -o-transition-property: transform;
```

Comment:
You should re-add this styling, it might avoid all the adjustments you needed to make in JavaScript

Reply:
You can do just `transition-property` too, the `o-` isn't necessary